src/v2/Apps/getAppRoutes.tsx

```diff
@@ -8,6 +8,7 @@ import { collectRoutes } from "v2/Apps/Collect/collectRoutes"
 import { consignRoutes } from "v2/Apps/Consign/consignRoutes"
 import { conversationRoutes } from "v2/Apps/Conversation/conversationRoutes"
 import { debugRoutes } from "./Debug/debugRoutes"
+import { exampleRoutes } from "./Example/exampleRoutes"
 import { fairRoutes } from "v2/Apps/Fair/fairRoutes"
```

Comment:
The only thing holding me up from merging is that this example route isn't under the v2 folder like most of these other apps. @damassi should it be?

Reply:
Ah, yup, should be under `v2/Apps` -- i'll move and merge

Reply:
Oh, this is just a relative vs absolute path thing -- its where it should be; full path resolves to `v2/Apps/Example`